router/appRest.js

```diff
@@ -99,10 +99,11 @@ function handler(req, res, channel, requestData) {
 								telemetryData.id = 'UNKNOWN_OPTION';
 								logData.botResponse =  responseKey;
 								response = literals.message[responseKey];
+								console.log('LOG : ' + JSON.stringify(logData))
```

Comment:
@ajinkyaghodake123 - User comma seprator to print each value in object , Don't user JSON stringy fy

Reply:
done